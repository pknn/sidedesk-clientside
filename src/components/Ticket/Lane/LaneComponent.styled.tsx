import styled from 'styled-components'

export const Basis = styled.div`
  flex-basis: 24%;
`
