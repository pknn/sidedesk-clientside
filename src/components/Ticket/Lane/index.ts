export { LaneComponent as Lane } from './LaneComponent'
