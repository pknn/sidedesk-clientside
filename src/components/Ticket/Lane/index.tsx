import React from 'react'

import { getMockTickets, ticketStatusOptions } from 'app/helpers/mockTicket'
import { LaneComponent } from './LaneComponent'

export const Lane = (): JSX.Element => {
  const tickets = getMockTickets(50)
  return (
    <div className="w-full flex justify-between">
      {ticketStatusOptions.map((ticketStatus) => (
        <LaneComponent
          key={ticketStatus}
          tickets={tickets.filter((ticket) => ticket.status === ticketStatus)}
          laneStatus={ticketStatus}
        />
      ))}
    </div>
  )
}
