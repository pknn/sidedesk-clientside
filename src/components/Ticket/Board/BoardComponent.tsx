import React from 'react'
import { DragDropContext, DropResult } from 'react-beautiful-dnd'

import { Lane } from 'app/components/Ticket/Lane'
import { Ticket, TicketStatus } from 'app/types/Ticket'
import { ticketStatusOptions } from 'app/helpers/mockTicket'
import { getKeyFromStatus } from 'app/helpers/statusMappers'

export interface DataProps {
  pendingTickets: Ticket[]
  acceptedTickets: Ticket[]
  resolvedTickets: Ticket[]
  rejectedTickets: Ticket[]
}

const toLaneComponent = (props: DataProps) =>
  ticketStatusOptions.map((ticketStatus: TicketStatus) => (
    <Lane
      key={ticketStatus}
      tickets={props[getKeyFromStatus(ticketStatus)]}
      laneStatus={ticketStatus}
    />
  ))

export const BoardComponent = (props: DataProps): JSX.Element => {
  const handleDragEnd = (result: DropResult) => {
    const { source, destination } = result
    console.log(source, destination)
  }

  return (
    <div className="w-full flex justify-between">
      <DragDropContext onDragEnd={handleDragEnd}>
        {toLaneComponent(props)}
      </DragDropContext>
    </div>
  )
}
