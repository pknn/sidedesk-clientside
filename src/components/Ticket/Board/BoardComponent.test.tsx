import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

import { getMockTickets } from 'app/helpers/mockTicket'

import { BoardComponent } from './BoardComponent'
import { LaneComponent } from '../Lane/LaneComponent'

describe('<BoardComponent />', () => {
  it('should render 4 lanes with correct status', () => {
    const mockTickets = getMockTickets(50)
    const shallowMountedComponent = shallow(
      <BoardComponent tickets={mockTickets} />,
    )

    expect(shallowMountedComponent.find(LaneComponent)).toHaveLength(4)
  })

  it('should render correctly', () => {
    const mockTickets = getMockTickets(50)
    const shallowMountedComponent = shallow(
      <BoardComponent tickets={mockTickets} />,
    )

    expect(toJson(shallowMountedComponent)).toMatchSnapshot()
  })
})
