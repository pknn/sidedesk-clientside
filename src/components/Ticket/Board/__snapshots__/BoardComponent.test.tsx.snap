// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BoardComponent /> should render correctly 1`] = `
<div
  className="w-full flex justify-between"
>
  <LaneComponent
    key="1"
    laneStatus={1}
    tickets={
      Array [
        Object {
          "createdAt": 1970-01-19T19:07:15.600Z,
          "description": "Ut aut itaque minus. Est ducimus tenetur blanditiis enim molestiae provident eum vel. Nemo consectetur quod cum ipsum aspernatur voluptatem magni est sapiente. Sequi debitis quibusdam quae ea veniam beatae in voluptate.",
          "id": 17872,
          "reporterEmail": "Hailey18@gmail.com",
          "reporterName": "Hailey Watsica",
          "status": 1,
          "title": "Placeat dolore sed et voluptates ipsum aut qui occaecati iste.",
          "updatedAt": 1970-01-19T19:07:15.600Z,
        },
        Object {
          "createdAt": 1970-01-19T19:05:49.200Z,
          "description": "Voluptatem saepe laborum animi consequatur voluptatum. Vel accusantium reprehenderit qui alias beatae. Repellat magnam deleniti et necessitatibus quia ut eveniet voluptas eius. Temporibus asperiores veniam.",
          "id": 33766,
          "reporterEmail": "Diego.Bogan@yahoo.com",
          "reporterName": "Diego Bogan",
          "status": 1,
          "title": "In aut aut ex quia vitae et alias necessitatibus vitae.",
          "updatedAt": 1970-01-19T19:05:49.200Z,
        },
        Object {
          "createdAt": 1970-01-19T19:07:15.600Z,
          "description": "Minus voluptatem ut soluta eos eos ut. Ipsa asperiores incidunt quam placeat excepturi. Et quae suscipit illum voluptatem qui autem. Iusto aut vero dolor velit voluptatum tempora architecto.",
          "id": 93918,
          "reporterEmail": "Alva.Ziemann72@hotmail.com",
          "reporterName": "Alva Ziemann",
          "status": 1,
          "title": "Non nesciunt praesentium sint illo dolores atque deleniti neque ipsam.",
          "updatedAt": 1970-01-19T19:07:15.600Z,
        },
        Object {
          "createdAt": 1970-01-19T19:07:15.600Z,
          "description": "Quis soluta expedita voluptatem. Dolore maiores sed ut nobis voluptatem incidunt. Eveniet nulla eligendi. Et est sit occaecati accusantium autem nobis.",
          "id": 43631,
          "reporterEmail": "Chester_Cole@yahoo.com",
          "reporterName": "Chester Cole",
          "status": 1,
          "title": "Dolor architecto possimus quod voluptatem odit possimus quia voluptatem tenetur.",
          "updatedAt": 1970-01-19T19:07:15.600Z,
        },
        Object {
          "createdAt": 1970-01-19T19:04:22.800Z,
          "description": "Excepturi qui non explicabo voluptas sit at enim. Et eum dignissimos ratione voluptatibus optio eligendi. Et provident corrupti magni aut. Ducimus quia esse mollitia autem nihil repellendus velit.",
          "id": 90220,
          "reporterEmail": "Jayda_Gerlach43@gmail.com",
          "reporterName": "Jayda Gerlach",
          "status": 1,
          "title": "Aut aperiam rem veniam optio culpa minima totam sunt possimus.",
          "updatedAt": 1970-01-19T19:04:22.800Z,
        },
        Object {
          "createdAt": 1970-01-19T19:07:15.600Z,
          "description": "Voluptatum harum voluptatem eaque voluptates est. Veritatis occaecati earum ullam vero temporibus. Nulla quibusdam qui rem dignissimos tenetur. Aut corporis animi illo aut impedit deleniti voluptatibus voluptatem.",
          "id": 72398,
          "reporterEmail": "Brian73@yahoo.com",
          "reporterName": "Brian Larkin",
          "status": 1,
          "title": "Ad quaerat minima molestias est facilis non rerum fugiat dicta.",
          "updatedAt": 1970-01-19T19:07:15.600Z,
        },
        Object {
          "createdAt": 1970-01-19T19:04:22.800Z,
          "description": "Totam iure quas aut non adipisci quisquam inventore officiis. Commodi possimus minima eum est modi natus pariatur. Fuga minima ipsum rem nesciunt non ipsa officiis. Ab qui voluptas eos ducimus laboriosam sed quis aut architecto.",
          "id": 8688,
          "reporterEmail": "Fabian.Wolf@gmail.com",
          "reporterName": "Fabian Wolf",
          "status": 1,
          "title": "Reiciendis voluptates consectetur ut quo laborum et nihil doloremque non.",
          "updatedAt": 1970-01-19T19:04:22.800Z,
        },
        Object {
          "createdAt": 1970-01-19T19:05:49.200Z,
          "description": "Culpa possimus non in. Voluptas est quibusdam quas numquam atque saepe. Animi dolores maiores possimus cum aut optio. Cum explicabo rerum ut id voluptatem est.",
          "id": 56674,
          "reporterEmail": "Ethyl_Stroman73@gmail.com",
          "reporterName": "Ethyl Stroman",
          "status": 1,
          "title": "Sapiente quia occaecati maxime et id nam ea dignissimos sit.",
          "updatedAt": 1970-01-19T19:05:49.200Z,
        },
        Object {
          "createdAt": 1970-01-19T19:04:22.800Z,
          "description": "Doloribus sequi magni quis ratione sed. Nihil quae qui et laudantium sequi dolorum quia fugiat vel. Dignissimos et ex neque sed eius. Et quasi eveniet et harum quia.",
          "id": 66524,
          "reporterEmail": "Kristin_Schmidt@yahoo.com",
          "reporterName": "Kristin Schmidt",
          "status": 1,
          "title": "Sint sit eum quia provident enim id optio laborum doloremque.",
          "updatedAt": 1970-01-19T19:04:22.800Z,
        },
        Object {
          "createdAt": 1970-01-19T19:07:15.600Z,
          "description": "Voluptatum fuga aliquid aut facilis nisi est at aut. Voluptate in rem dolore. Reiciendis ea rerum aut nostrum ipsam dignissimos cumque quia nulla. Cumque maxime dolore quasi.",
          "id": 23580,
          "reporterEmail": "Bennie41@yahoo.com",
          "reporterName": "Bennie Hessel",
          "status": 1,
          "title": "Quos in ducimus optio aut est quis officiis ut tempora.",
          "updatedAt": 1970-01-19T19:07:15.600Z,
        },
        Object {
          "createdAt": 1970-01-19T19:07:15.600Z,
          "description": "Beatae et voluptatem. Officiis aut esse illum expedita dicta nulla est. Dolores vel quo quis qui qui fugiat illo eveniet. Accusamus omnis veniam.",
          "id": 48113,
          "reporterEmail": "Reginald59@gmail.com",
          "reporterName": "Reginald Pagac",
          "status": 1,
          "title": "Est sint magnam consectetur esse exercitationem exercitationem aspernatur qui fugiat.",
          "updatedAt": 1970-01-19T19:07:15.600Z,
        },
        Object {
          "createdAt": 1970-01-19T19:04:22.800Z,
          "description": "Necessitatibus atque nemo mollitia consequatur quis eum optio est. Beatae aliquid perspiciatis doloribus ratione omnis. Unde ea in sed et iusto ratione aut. Cupiditate ipsa eveniet consectetur saepe est.",
          "id": 89926,
          "reporterEmail": "Saul.Turner@yahoo.com",
          "reporterName": "Saul Turner",
          "status": 1,
          "title": "Libero est voluptatum minima voluptas quo blanditiis numquam blanditiis corrupti.",
          "updatedAt": 1970-01-19T19:04:22.800Z,
        },
      ]
    }
  />
  <LaneComponent
    key="2"
    laneStatus={2}
    tickets={
      Array [
        Object {
          "createdAt": 1970-01-19T19:04:22.800Z,
          "description": "In ut facilis incidunt. Nostrum suscipit molestias labore ad. Unde sed sunt. Libero praesentium esse praesentium debitis expedita quia nulla.",
          "id": 7532,
          "reporterEmail": "Jalen.Rutherford20@yahoo.com",
          "reporterName": "Jalen Rutherford",
          "status": 2,
          "title": "Repudiandae accusantium et nihil aut in animi numquam quis laborum.",
          "updatedAt": 1970-01-19T19:04:22.800Z,
        },
        Object {
          "createdAt": 1970-01-19T19:04:22.800Z,
          "description": "Ipsa ipsum ut inventore expedita blanditiis molestias enim suscipit nam. Est consequatur nostrum nemo. Eaque voluptatem dolor corrupti porro quo fuga. Deserunt repellendus dignissimos hic ad.",
          "id": 21913,
          "reporterEmail": "Maryse_Jones@gmail.com",
          "reporterName": "Maryse Jones",
          "status": 2,
          "title": "Et qui dolorem pariatur quo cum delectus eos qui occaecati.",
          "updatedAt": 1970-01-19T19:04:22.800Z,
        },
        Object {
          "createdAt": 1970-01-19T19:04:22.800Z,
          "description": "Nihil consequatur corrupti consectetur labore expedita cum architecto laboriosam. Consectetur molestiae dolores et. Autem consectetur fugiat eum aut aperiam rerum sit aut expedita. Est voluptatem ducimus eligendi ex animi impedit esse voluptates eos.",
          "id": 79172,
          "reporterEmail": "Laverne_Schmeler@yahoo.com",
          "reporterName": "Laverne Schmeler",
          "status": 2,
          "title": "Porro consequatur voluptate et sapiente ut laboriosam magni eos labore.",
          "updatedAt": 1970-01-19T19:04:22.800Z,
        },
        Object {
          "createdAt": 1970-01-19T19:07:15.600Z,
          "description": "Eos omnis explicabo. Asperiores voluptate numquam. Blanditiis qui neque fugiat quia. Quidem eos quos et nesciunt et est.",
          "id": 10337,
          "reporterEmail": "Alfred_Rohan@yahoo.com",
          "reporterName": "Alfred Rohan",
          "status": 2,
          "title": "Suscipit quia sint excepturi eligendi libero quas possimus adipisci libero.",
          "updatedAt": 1970-01-19T19:07:15.600Z,
        },
        Object {
          "createdAt": 1970-01-19T19:05:49.200Z,
          "description": "Dicta ea sunt exercitationem aut at. Qui cum ut qui neque quia quia nostrum aperiam quasi. Quaerat deserunt dolor. Sed quae ad nemo est quo labore ipsum.",
          "id": 36600,
          "reporterEmail": "Vince17@hotmail.com",
          "reporterName": "Vince Lehner",
          "status": 2,
          "title": "Eaque sed sint sapiente tenetur perferendis repellendus eveniet numquam quia.",
          "updatedAt": 1970-01-19T19:05:49.200Z,
        },
        Object {
          "createdAt": 1970-01-19T19:05:49.200Z,
          "description": "Quas tempore similique. Earum temporibus a quia at quam. Et est repellendus corporis expedita beatae excepturi. Autem eum laudantium qui iste perferendis consectetur facere in quis.",
          "id": 35014,
          "reporterEmail": "Jonatan_McKenzie84@hotmail.com",
          "reporterName": "Jonatan McKenzie",
          "status": 2,
          "title": "Aliquid velit neque sunt optio officiis sit odit doloremque voluptas.",
          "updatedAt": 1970-01-19T19:05:49.200Z,
        },
        Object {
          "createdAt": 1970-01-19T19:07:15.600Z,
          "description": "Tempora rerum voluptates ut aut odit blanditiis cupiditate dolores. Et aut itaque laborum qui necessitatibus hic sed doloribus consequatur. Animi aperiam fugiat quia blanditiis est fugiat placeat. Nostrum voluptas consequuntur aliquam nesciunt.",
          "id": 53494,
          "reporterEmail": "David_Rosenbaum23@gmail.com",
          "reporterName": "David Rosenbaum",
          "status": 2,
          "title": "Et eum aut voluptatem velit accusamus omnis beatae sint velit.",
          "updatedAt": 1970-01-19T19:07:15.600Z,
        },
        Object {
          "createdAt": 1970-01-19T19:04:22.800Z,
          "description": "Aut voluptate aut sed magnam aliquid doloribus. Occaecati vel fugiat excepturi iusto eum. Et quod ratione suscipit inventore est. Praesentium optio non quibusdam corrupti quam dolor accusamus.",
          "id": 38986,
          "reporterEmail": "Brielle_Collier@gmail.com",
          "reporterName": "Brielle Collier",
          "status": 2,
          "title": "Dolorem dolor ut est modi quasi sunt dolores cumque sit.",
          "updatedAt": 1970-01-19T19:04:22.800Z,
        },
        Object {
          "createdAt": 1970-01-19T19:07:15.600Z,
          "description": "Harum quaerat laudantium quod molestiae voluptatem quaerat sit porro. At optio aut delectus. Sed error vero sint omnis necessitatibus sapiente ullam. Necessitatibus architecto sint quam sed sed reprehenderit assumenda voluptates inventore.",
          "id": 34150,
          "reporterEmail": "Brigitte84@hotmail.com",
          "reporterName": "Brigitte Rowe",
          "status": 2,
          "title": "Fugit cupiditate adipisci numquam quo numquam aspernatur sed quia velit.",
          "updatedAt": 1970-01-19T19:07:15.600Z,
        },
        Object {
          "createdAt": 1970-01-19T19:07:15.600Z,
          "description": "Earum assumenda veritatis non rem incidunt ut corrupti. Sint porro et. Excepturi illum minus omnis eos. Nam eum ab dolorum maiores vel sed.",
          "id": 81062,
          "reporterEmail": "Johnnie.Denesik43@gmail.com",
          "reporterName": "Johnnie Denesik",
          "status": 2,
          "title": "Quasi et numquam ipsam omnis beatae corrupti neque non deleniti.",
          "updatedAt": 1970-01-19T19:07:15.600Z,
        },
        Object {
          "createdAt": 1970-01-19T19:04:22.800Z,
          "description": "Tempore aut odit qui omnis sequi provident enim. Eos quos sequi magnam debitis vitae. Architecto doloremque veniam non omnis. Unde commodi recusandae aut autem.",
          "id": 15317,
          "reporterEmail": "Isidro.Carroll85@hotmail.com",
          "reporterName": "Isidro Carroll",
          "status": 2,
          "title": "Fugiat quo adipisci et eius quis sequi quis eos deleniti.",
          "updatedAt": 1970-01-19T19:04:22.800Z,
        },
        Object {
          "createdAt": 1970-01-19T19:07:15.600Z,
          "description": "Voluptate tenetur labore labore fugiat. Dignissimos et quam qui. Sunt a modi voluptatibus est est blanditiis rem non aut. Voluptatem soluta nobis.",
          "id": 67004,
          "reporterEmail": "Cleo.Olson95@gmail.com",
          "reporterName": "Cleo Olson",
          "status": 2,
          "title": "Dolore iusto rem magnam eveniet et iusto sed corrupti omnis.",
          "updatedAt": 1970-01-19T19:07:15.600Z,
        },
        Object {
          "createdAt": 1970-01-19T19:04:22.800Z,
          "description": "Fugiat aliquam excepturi ut nihil repellat beatae sint. Fuga et mollitia nisi ullam repellat. Et officia repellendus aut inventore beatae blanditiis. Magni perspiciatis eos voluptas reprehenderit libero.",
          "id": 28694,
          "reporterEmail": "Samson.Steuber20@gmail.com",
          "reporterName": "Samson Steuber",
          "status": 2,
          "title": "Quae aut fugit non iure sequi eum voluptatem quidem vitae.",
          "updatedAt": 1970-01-19T19:04:22.800Z,
        },
      ]
    }
  />
  <LaneComponent
    key="3"
    laneStatus={3}
    tickets={
      Array [
        Object {
          "createdAt": 1970-01-19T19:07:15.600Z,
          "description": "Aut est harum sunt ut deserunt cupiditate dolor ut. Sint nisi recusandae iste. Dolores saepe architecto quae omnis. Cupiditate deleniti qui libero optio.",
          "id": 76707,
          "reporterEmail": "Flavie.Feeney72@hotmail.com",
          "reporterName": "Flavie Feeney",
          "status": 3,
          "title": "Alias dolorum cumque aut mollitia culpa earum necessitatibus quibusdam voluptatem.",
          "updatedAt": 1970-01-19T19:07:15.600Z,
        },
        Object {
          "createdAt": 1970-01-19T19:04:22.800Z,
          "description": "Asperiores doloribus alias rerum consectetur id. Vel est dolorem laboriosam facere dignissimos minima. Earum repellat quam voluptatem error corporis quis illum. Est perferendis tempora rerum saepe in et.",
          "id": 12105,
          "reporterEmail": "Jessie_Wiza32@yahoo.com",
          "reporterName": "Jessie Wiza",
          "status": 3,
          "title": "Necessitatibus eaque nihil vel odio omnis accusamus doloremque est reiciendis.",
          "updatedAt": 1970-01-19T19:04:22.800Z,
        },
        Object {
          "createdAt": 1970-01-19T19:05:49.200Z,
          "description": "Reprehenderit deleniti nobis minima amet. Eos veritatis eaque fugit tempora. Rerum sunt assumenda repellat alias exercitationem eveniet quos sit officiis. Aut voluptas ea omnis voluptates saepe alias.",
          "id": 7517,
          "reporterEmail": "Zoey.Torphy2@hotmail.com",
          "reporterName": "Zoey Torphy",
          "status": 3,
          "title": "Ut sint et non aut reiciendis voluptas qui est magni.",
          "updatedAt": 1970-01-19T19:05:49.200Z,
        },
        Object {
          "createdAt": 1970-01-19T19:05:49.200Z,
          "description": "Deserunt suscipit sapiente beatae. Iusto nisi sequi. Et rerum est quis voluptatem harum. Molestias voluptatem corporis.",
          "id": 60247,
          "reporterEmail": "Montana.Jast@yahoo.com",
          "reporterName": "Montana Jast",
          "status": 3,
          "title": "Aut adipisci accusantium quibusdam voluptate delectus eligendi ex blanditiis sequi.",
          "updatedAt": 1970-01-19T19:05:49.200Z,
        },
        Object {
          "createdAt": 1970-01-19T19:07:15.600Z,
          "description": "Ipsum facere omnis tempore eum tenetur ea possimus ab. Doloremque et quisquam consequatur eos ut laudantium omnis. Illo veniam sed aut iste culpa harum facilis et nihil. Cum dolor quam voluptatem deleniti est quia.",
          "id": 35053,
          "reporterEmail": "Gladys.Rosenbaum29@hotmail.com",
          "reporterName": "Gladys Rosenbaum",
          "status": 3,
          "title": "Aliquid facilis eligendi modi ut hic pariatur sit aut vitae.",
          "updatedAt": 1970-01-19T19:07:15.600Z,
        },
        Object {
          "createdAt": 1970-01-19T19:04:22.800Z,
          "description": "Laboriosam quas ut iusto unde rerum id debitis corrupti. Quia tempora magni blanditiis quibusdam. Voluptas voluptatem temporibus assumenda. Maiores sint possimus.",
          "id": 24772,
          "reporterEmail": "Bobbie.Cruickshank@yahoo.com",
          "reporterName": "Bobbie Cruickshank",
          "status": 3,
          "title": "Sint ad voluptatem et et ut ratione qui voluptatem quia.",
          "updatedAt": 1970-01-19T19:04:22.800Z,
        },
        Object {
          "createdAt": 1970-01-19T19:05:49.200Z,
          "description": "Consequatur ab labore provident quo commodi. Aspernatur fugit et eum consequatur quia vero voluptas. Itaque ea beatae praesentium et soluta. Sint recusandae incidunt illum laboriosam deserunt quo laborum qui.",
          "id": 16436,
          "reporterEmail": "Rupert.Murray@yahoo.com",
          "reporterName": "Rupert Murray",
          "status": 3,
          "title": "Aut eos distinctio dignissimos qui tempore sit ea qui voluptate.",
          "updatedAt": 1970-01-19T19:05:49.200Z,
        },
        Object {
          "createdAt": 1970-01-19T19:07:15.600Z,
          "description": "Saepe est recusandae libero et porro voluptas repellat laudantium. Deserunt at libero. Dolor similique et harum facilis voluptatem quisquam. Molestiae sed excepturi.",
          "id": 53940,
          "reporterEmail": "Millie_Volkman@gmail.com",
          "reporterName": "Millie Volkman",
          "status": 3,
          "title": "In aut cumque distinctio exercitationem aperiam debitis et quo et.",
          "updatedAt": 1970-01-19T19:07:15.600Z,
        },
        Object {
          "createdAt": 1970-01-19T19:04:22.800Z,
          "description": "Consequuntur a sunt eaque. Corrupti reiciendis reprehenderit ab molestiae. Aspernatur similique quo qui explicabo repudiandae. Quis sed rerum pariatur in.",
          "id": 16586,
          "reporterEmail": "Cindy2@hotmail.com",
          "reporterName": "Cindy Kuhn",
          "status": 3,
          "title": "Est rem eos praesentium totam soluta voluptatem voluptas et sit.",
          "updatedAt": 1970-01-19T19:04:22.800Z,
        },
        Object {
          "createdAt": 1970-01-19T19:05:49.200Z,
          "description": "Dolor aspernatur reprehenderit ad dolore deleniti sapiente magni in omnis. Earum error magni fugiat est odio harum laboriosam. Saepe natus doloribus itaque sit et quos quas minus ut. Pariatur quasi totam.",
          "id": 37351,
          "reporterEmail": "Lilla.McGlynn@yahoo.com",
          "reporterName": "Lilla McGlynn",
          "status": 3,
          "title": "Qui inventore ut enim odio sit eum reprehenderit cupiditate recusandae.",
          "updatedAt": 1970-01-19T19:05:49.200Z,
        },
        Object {
          "createdAt": 1970-01-19T19:05:49.200Z,
          "description": "Possimus maiores veritatis id ab fugiat et consequuntur. Accusantium est ut voluptas non enim neque. Repellat libero sequi modi eos neque omnis iusto. Eaque enim alias doloremque exercitationem aut id molestias.",
          "id": 5213,
          "reporterEmail": "Darian_Okuneva@gmail.com",
          "reporterName": "Darian Okuneva",
          "status": 3,
          "title": "Commodi atque quia asperiores rem et qui provident molestiae ad.",
          "updatedAt": 1970-01-19T19:05:49.200Z,
        },
        Object {
          "createdAt": 1970-01-19T19:05:49.200Z,
          "description": "Aut quasi dolor sequi omnis rerum vero saepe. At quo voluptates delectus id. Sed veniam quia iusto eum sunt veritatis. Ipsam et quibusdam sequi sed rerum id non porro.",
          "id": 65062,
          "reporterEmail": "Kennedy81@hotmail.com",
          "reporterName": "Kennedy Kemmer",
          "status": 3,
          "title": "Molestiae aut eveniet esse repellendus dolores quia praesentium et deleniti.",
          "updatedAt": 1970-01-19T19:05:49.200Z,
        },
      ]
    }
  />
  <LaneComponent
    key="4"
    laneStatus={4}
    tickets={
      Array [
        Object {
          "createdAt": 1970-01-19T19:05:49.200Z,
          "description": "Occaecati officia unde impedit itaque placeat quas veniam quidem et. Dolores impedit a unde officia et impedit. Accusamus maiores minus atque dolor qui. Iste facere incidunt suscipit.",
          "id": 51117,
          "reporterEmail": "Richard_Rodriguez62@yahoo.com",
          "reporterName": "Richard Rodriguez",
          "status": 4,
          "title": "Vel reiciendis excepturi rem omnis repudiandae odit quis nulla veritatis.",
          "updatedAt": 1970-01-19T19:05:49.200Z,
        },
        Object {
          "createdAt": 1970-01-19T19:05:49.200Z,
          "description": "Pariatur qui possimus. Est eum et. Omnis vel necessitatibus. Voluptatem sed nam.",
          "id": 81135,
          "reporterEmail": "Jennings.Cassin@hotmail.com",
          "reporterName": "Jennings Cassin",
          "status": 4,
          "title": "Iure deleniti unde eos dolorem praesentium dolorem qui laborum quae.",
          "updatedAt": 1970-01-19T19:05:49.200Z,
        },
        Object {
          "createdAt": 1970-01-19T19:04:22.800Z,
          "description": "Ipsa et dolorem voluptas non aut in ut mollitia accusamus. Ipsum minus reprehenderit perferendis doloribus. Maxime occaecati eum dolores est placeat excepturi modi. Neque repellendus repudiandae iste.",
          "id": 11223,
          "reporterEmail": "Gay_Kling@yahoo.com",
          "reporterName": "Gay Kling",
          "status": 4,
          "title": "Soluta qui sunt nihil iusto recusandae sit iusto et omnis.",
          "updatedAt": 1970-01-19T19:04:22.800Z,
        },
        Object {
          "createdAt": 1970-01-19T19:05:49.200Z,
          "description": "Eveniet sint repellendus sint amet laborum voluptate esse. Voluptatibus in quaerat nulla et animi ut. Et voluptatem et voluptas. Provident non rem ut eius sit sequi.",
          "id": 64067,
          "reporterEmail": "Brennan_Satterfield56@hotmail.com",
          "reporterName": "Brennan Satterfield",
          "status": 4,
          "title": "Porro aut qui maxime quas aliquam et atque quidem numquam.",
          "updatedAt": 1970-01-19T19:05:49.200Z,
        },
        Object {
          "createdAt": 1970-01-19T19:07:15.600Z,
          "description": "Et commodi voluptas quia ut voluptatum magnam distinctio. Nemo debitis et tempora ea eaque sint aut voluptas. Et inventore aliquam fugiat aperiam perferendis architecto et et. Veritatis quo dolores nulla.",
          "id": 54153,
          "reporterEmail": "Kyle33@yahoo.com",
          "reporterName": "Kyle Stoltenberg",
          "status": 4,
          "title": "Sequi amet voluptas tempora fugiat nam doloribus sapiente enim voluptates.",
          "updatedAt": 1970-01-19T19:07:15.600Z,
        },
        Object {
          "createdAt": 1970-01-19T19:04:22.800Z,
          "description": "Quos fugiat quod quia autem a. Dolorem est possimus est ut corporis nam in ut inventore. Odit animi voluptatem minus. Eos corporis quibusdam.",
          "id": 86392,
          "reporterEmail": "Justina85@hotmail.com",
          "reporterName": "Justina Ondricka",
          "status": 4,
          "title": "Ea atque est id ut debitis voluptas tempora sed voluptatibus.",
          "updatedAt": 1970-01-19T19:04:22.800Z,
        },
        Object {
          "createdAt": 1970-01-19T19:05:49.200Z,
          "description": "Aperiam voluptatum et voluptate labore. Dolore eveniet in porro dignissimos quo aut. Qui molestias alias alias debitis suscipit sunt. Iusto facilis sit tempore velit repellat ut modi doloribus.",
          "id": 21074,
          "reporterEmail": "Chris_Jakubowski24@gmail.com",
          "reporterName": "Chris Jakubowski",
          "status": 4,
          "title": "Possimus voluptatem sapiente omnis error ipsa rerum magnam sed eum.",
          "updatedAt": 1970-01-19T19:05:49.200Z,
        },
        Object {
          "createdAt": 1970-01-19T19:04:22.800Z,
          "description": "Minus repudiandae ab aut molestiae sit ut voluptatem tempore. Consequatur molestias maiores ducimus illo quia. Sint omnis debitis sunt facilis. Nihil aut eligendi quasi aspernatur harum delectus minus aspernatur.",
          "id": 99824,
          "reporterEmail": "Vada.Goodwin14@hotmail.com",
          "reporterName": "Vada Goodwin",
          "status": 4,
          "title": "Nihil enim eum eos molestias fugit tempore temporibus aliquid expedita.",
          "updatedAt": 1970-01-19T19:04:22.800Z,
        },
        Object {
          "createdAt": 1970-01-19T19:05:49.200Z,
          "description": "Unde perspiciatis in ut aut aut et voluptatem cum nisi. Dolores impedit voluptatum autem minima sit. Non numquam suscipit. Aut qui quidem velit.",
          "id": 88754,
          "reporterEmail": "Gennaro_Lubowitz@hotmail.com",
          "reporterName": "Gennaro Lubowitz",
          "status": 4,
          "title": "Deserunt placeat necessitatibus vel qui autem aut corporis corporis dolorem.",
          "updatedAt": 1970-01-19T19:05:49.200Z,
        },
        Object {
          "createdAt": 1970-01-19T19:04:22.800Z,
          "description": "Perferendis ipsam doloremque. Et iusto animi dolor. Similique sed perferendis optio qui quaerat alias. Voluptatum magni quam non.",
          "id": 25504,
          "reporterEmail": "Donny_Brekke@hotmail.com",
          "reporterName": "Donny Brekke",
          "status": 4,
          "title": "Numquam nam possimus est molestiae ratione doloribus aut molestias nostrum.",
          "updatedAt": 1970-01-19T19:04:22.800Z,
        },
        Object {
          "createdAt": 1970-01-19T19:05:49.200Z,
          "description": "Cupiditate et dolor nihil eos. Est enim est labore quo et et voluptatem autem qui. Sint odio magnam accusamus in. Qui vero vel quo et.",
          "id": 51562,
          "reporterEmail": "Wallace.Adams66@yahoo.com",
          "reporterName": "Wallace Adams",
          "status": 4,
          "title": "Nemo asperiores qui earum rem quaerat aliquid dolorem nobis iure.",
          "updatedAt": 1970-01-19T19:05:49.200Z,
        },
        Object {
          "createdAt": 1970-01-19T19:07:15.600Z,
          "description": "Eius exercitationem et id et est. Molestias voluptatem repellendus quis. Eos accusamus laudantium. Reprehenderit quis ea vitae consequuntur quia sed odio.",
          "id": 2295,
          "reporterEmail": "Myrtis40@gmail.com",
          "reporterName": "Myrtis Kihn",
          "status": 4,
          "title": "Aut dolorem eum et sunt cum totam nesciunt omnis qui.",
          "updatedAt": 1970-01-19T19:07:15.600Z,
        },
        Object {
          "createdAt": 1970-01-19T19:05:49.200Z,
          "description": "Et totam est. Rerum ut ut nesciunt consequuntur consequatur quam animi et sapiente. Fuga fuga dignissimos possimus vitae qui maxime provident impedit labore. Nemo et explicabo pariatur quas itaque excepturi quia est.",
          "id": 36819,
          "reporterEmail": "Greta92@hotmail.com",
          "reporterName": "Greta Smith",
          "status": 4,
          "title": "Eius et cupiditate quidem voluptatem cum exercitationem illum quia deserunt.",
          "updatedAt": 1970-01-19T19:05:49.200Z,
        },
      ]
    }
  />
</div>
`;
